ports/espressif/boards/adafruit_qualia_s3_rgb666/mpconfigboard.mk

```diff
@@ -14,3 +14,5 @@ CIRCUITPY_ESP_PSRAM_MODE = opi
 CIRCUITPY_ESP_PSRAM_FREQ = 80m
 
 CIRCUITPY_DOTCLOCKFRAMEBUFFER = 1
+
+CIRCUITPY__EVE = 1
```

Comment:
This board is already for driving a particular kind of display, so let's not add `_eve` to this.

Reply:
I removed this line from espressif/boards/adafruit_qualia_s3_rgb666/mpconfigboard.mk.
The configuration is refactored to espressif/mpconfigport.mk at commit https://github.com/adafruit/circuitpython/commit/8d578670be362d3f462a5dda2bd5ed6fb290c50c